libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -684,6 +820,9 @@ ebpf_free_timer_work_item(_Frees_ptr_opt_ ebpf_timer_work_item_t* work_item)
 
     WaitForThreadpoolTimerCallbacks(work_item->threadpool_timer, true);
     CloseThreadpoolTimer(work_item->threadpool_timer);
+    for (auto& dpc : _ebpf_emulated_dpcs) {
```

Comment:
I dont understand why is this needed here?

Reply:
Epoch code relies on a KeFlushDpc call when freeing a timer. This is to provide similar functionality.